gui/doc/table.md

```diff
@@ -38,155 +58,296 @@ tables:
+to add the [*show_all](#p-show_all):
+
+!!! example "Page content"
     === "Markdown"
+        ```
+        <|{data}|table|show_all|>
+        ```
+    === "HTML"
+        ```html
+        <taipy:table data="{data}" show_all="true"/>
+        ```
+
+Now the table displays all the data rows, and the scrollbar lets you navigate among all of
+them:
 
+<figure class="tp-center">
+    <img src="../table-show_all-d.png" class="visible-dark"  width="50%"/>
+    <img src="../table-show_all-l.png" class="visible-light" width="50%"/>
+    <figcaption>Showing all the rows (partial)</figcaption>
+</figure>
+
+Setting the [*allow_all_rows*](#p-allow_all_rows) property to True for a paginated table
+adds the 'All' option to the page size options, so the user can switch from one mode to
+the other.
+
+### Show specific columns
+
+If you want to display a specific set of columns, you can use the [*columns*](#p-columns)
+property to indicate what columns should be displayed.
+
+Here is how you would define the table control if you want to hide the column *y2*
+from the examples above:
+
+!!! example "Page content"
+    === "Markdown"
         ```
-        <|{data}|table|columns=Col 1;Col 2;Col 3|page_size=10|page_size_options=10;30;100|date_format=eee dd MMM yyyy|not allow_all_rows|show_all=No|auto_loading=False|width=100vw|height=100vw|selected={selection}|>
+        <|{data}|table|columns=x;y1|>
         ```
 
     === "HTML"
-
         ```html
-        <taipy:table columns="Col 1;Col 2;Col 3" page_size="10" page_size_options="10;30;100" date_format="eee dd MMM yyyy" allow_all_rows="False" show_all="False" auto_loading="False" width="100vw" height="100vw" selected="{selection}">{data}</taipy:table>
+        <taipy:table columns="x;y1">{data}</taipy:table>
         ```
 
-### Aggregation
+And the *y2* column is not displayed any longer:
+<figure class="tp-center">
+    <img src="../table-columns-d.png" class="visible-dark"  width="50%"/>
+    <img src="../table-columns-l.png" class="visible-light" width="50%"/>
+    <figcaption>Specifying the visible columns</figcaption>
+</figure>
 
-To get the aggregation functionality in your table, you must indicate which columns can be aggregated, and
-how to perform the aggregation.
+### Styling rows
 
-This is done using the _group_by_ and _apply_ properties.
+To give a specific style to a table row, you will use the [*style*](#p-style) property.<br/>
+This property holds a function that is invoked when each row is rendered, and it must return
+the name of a style, defined in CSS.
 
-The _group_by[column_name]_ property, when set to _True_ indicates that the column _column_name_ can be
-aggregated.
+Here is how a row styling function can be defined:
 
-The function provided in the _apply[column_name]_ property indicates how to perform this aggregation.
-The value of this property, which is a string, can be:
+```py
+def even_odd_style(_1, index, _2):
```

Comment:
I think you can omit the 3rd parameter (_2)

Reply:
I think you are right. It is a bit clearer as such in my opinion, with respect to the documentation of the function signature.